lib/tasks/gather.js

```diff
@@ -197,6 +197,21 @@ class TaskGather extends SttTask {
       throw new Error('Cobalt requires a model to be specified');
     }
 
+    // actionHook delay
+    this._actionHookDelayEnabled = cs.actionHookDelayEnabled ||
+      (this.actionHookDelayAction !== null && this.actionHookDelayAction !== undefined);
```

Comment:
is it necessary to test against both null and undefined?  Please research if this gives the same result
```
this._actionHookDelayEnabled = !!cs.actionHookDelayEnabled 
```

Reply:
followed suggestion